src/axom/spin/internal/linear_bvh/TraversalPredicates.hpp

```diff
@@ -97,9 +98,9 @@ class TraversalPredicates
   template < typename RayType >
   AXOM_HOST_DEVICE
   static inline bool rayIntersectsLeftBin(
-      const RayType& r, const vec4_t& s1, const vec4_t& s2,
-      FloatType TOL=numerics::floating_point_limits<FloatType>::epsilon()
-      ) noexcept;
+    const RayType& r, const vec4_t& s1, const vec4_t& s2,
+    FloatType TOL=numerics::floating_point_limits<FloatType>::epsilon()
```

Comment:
I think I have mentioned this before, but I think the tolerance should be relative so either the AABB bounds or the total "scene" bounds.

Reply:
Yes, those are default arguments. At the call site we can pass a more appropriate tolerance. We are currently not doing that, but I think all the machinery is there for us to do so. I'll create a Github issue for this.

Reply:
See #221